bootstrap/kubeadm/internal/bottlerocket/bottlerocket.go

```diff
@@ -27,6 +28,7 @@ type BottlerocketConfig struct {
 	ProxyConfiguration          bootstrapv1.ProxyConfiguration
 	RegistryMirrorConfiguration bootstrapv1.RegistryMirrorConfiguration
 	KubeletExtraArgs            map[string]string
+	Taints			            []corev1.Taint
```

Comment:
Is this a weird indentation where it used tab instead of spaces?

Reply:
huh weird this looked OK on my local